doc/rbd/rbd-openstack.rst

```diff
@@ -221,6 +221,20 @@ For more information about the configuration options available in Glance please
 .. important:: Glance has not completely moved to 'store' yet.
     So we still need to configure the store in the DEFAULT section.
 
+Mitaka
+~~~~~~
+
+For Openstack Mitaka and Ceph Jewel we need to configure store in the ``[glance_store]`` section::
```

Comment:
I would recommend just re-using the first sentence from the Juno section: "Edit ``/etc/glance/glance-api.conf`` and add under the ``[glance_store]`` section::"